nvflare/private/fed/app/fl_conf.py

```diff
@@ -92,6 +92,10 @@ def process_config_element(self, config_ctx: ConfigContext, node: Node):
             self.enable_byoc = element
             return
 
+        if path == "config_validator" and isinstance(element, dict):
```

Comment:
Let's call it app_validator since it could go beyond just config files. Also the concept should be "app", not "config" which is too generic.

The provision also needs to change to call it "app_validator".